xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -463,9 +463,12 @@ void CDVDSubtitlesLibass::ApplyStyle(const std::shared_ptr<struct style>& subSty
     }
 
     // ass_set_line_spacing do not scale, so we have to scale to frame size
-    ass_set_line_spacing(m_renderer,
+    if (m_subtitleType != NATIVE)
```

Comment:
do this looks wrong,
moreover on master branch preventing to call ass_set_line_spacing will broke the line space setting

Reply:
you should provide some more explanations about this,
is an attempt to workaround a libass library bug? or do you think there is another problem?